src/toil/cwl/cwltoil.py

```diff
@@ -192,6 +192,7 @@ def __init__(self, tool, cwljob):
         #super(CWLJob, self).__init__()
         self.cwltool = tool
         self.cwljob = cwljob
+        self.executor_options = kwargs
```

Comment:
Sorry, scratch all that. I read further down in your PR and it is better the way it is.